CodeForAfricaLabs/CitizenReporter.Android
# Ft push notification

#### What does this PR do?
Allow users to receive push notification.
#### Description of Task to be completed?
Send push notifications to all clients(messenger, android, iOS) consuming the API on creation of a new assignment.
#### How should this be manually tested?
After cloning the repo and running the app on Android studio:
- Ensure the app is running on the background, go to the logcat on Android studio and copy the registration token displayed
- Go to the Firebase Cloud Messenger: https://console.firebase.google.com/project/storied-shell-179008/notification
- Go to the Notifications section, click on send message, enter any message text, then paste the registration token to the single device target.
- Click Send Message button. You should receive a notification on your Android device with the message you typed on the Firebase Cloud Messenger console
#### Any background context you want to provide?
Make sure you device has google playstore installed.

![push_notification image](https://user-images.githubusercontent.com/25563841/30431676-0a9fac32-9968-11e7-8851-69e73e661a0e.png)
